src/fixup/fixup.cpp

```diff
@@ -717,6 +717,13 @@ TaskStatus ApplyFloors(T *rc) {
 }
 
 template TaskStatus ApplyFloors<MeshBlockData<Real>>(MeshBlockData<Real> *rc);
+template <>
+TaskStatus ApplyFloors<MeshData<Real>>(MeshData<Real> *rc) {
```

Comment:
I wonder if the specialization isn't working properly. Maybe worth double checking that this version of the code is actually being called. (And same for the others...)